apps/rush-lib/src/schemas/build-cache.schema.json

```diff
@@ -27,7 +39,9 @@
             "cacheProvider": {
               "type": "string",
               "enum": ["local-only"]
-            }
+            },
+
+            "cacheEntryNamePattern": { "$ref": "#/definitions/anything" }
```

Comment:
Confused... why is this the `anything` definition and not a string as above?

Reply:
Simplicity. We've done this in other places too. For example: https://github.com/microsoft/rushstack/blob/master/apps/rush-lib/src/schemas/command-line.schema.json#L90-L100

Because each of these has a `"additionalProperties": false`, we need to include dummy entries for all of the shared fields.